optax/_src/transform.py

```diff
@@ -76,17 +76,39 @@ def update_fn(updates, state, params=None):
 
 
 def _update_moment(updates, moments, decay, order):
-  """Compute the exponential moving average of the `order-th` moment."""
+  """Compute the exponential moving average of the `order`-th moment."""
   return jax.tree_multimap(
       lambda g, t: (1 - decay) * (g ** order) + decay * t, updates, moments)
 
 
+def _update_moment_norm(updates, moments, decay, order):
```

Comment:
I find it slightly confusing because "norm" can be interpreted as a vector/matrix norm. Perhaps `_update_moment_per_elem_norm` would fit better?